e2e/ui/utils/client-utils.js

```diff
@@ -1,6 +1,8 @@
 const axios = require('axios');
 const nconf = require('nconf');
 
+const delay = timeout => new Promise(resolve => setTimeout(resolve, timeout));
```

Comment:
why it is added?

Reply:
I just noticed there was an error message while running the tests, so I fixed it. 
It's just a small bug fix, not really related to zap or glue